doc/sphinx/language/extensions/implicit-arguments.rst

```diff
@@ -420,6 +420,14 @@ but succeeds in
 Deactivation of implicit arguments for parsing
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. insertprodn term_explicit term_explicit
+
+.. prodn::
+   term_explicit ::= @ @qualid_annotated
+
+This syntax can be used to disable implicit arguments locally.  To
```

Comment:
Can we be more specific on the scope than "locally"?  I was thinking something like "within the term" or "with the qualid_annotated", but qualid_annotated can't contain any function applications, so not sure what I'd say.

Reply:
I'll explain better and put an example.